reactor-pool/src/main/java/reactor/pool/SimpleDequePool.java

```diff
@@ -469,7 +475,7 @@ private Mono<POOLABLE> allocatorWithScheduler() {
 		return poolConfig.allocator();
 	}
 
-	Mono<Void> warmupMono(int index, int max, long startWarmupIteration, Mono<POOLABLE> allocator) {
+	Mono<POOLABLE> warmupMono(int index, int max, long startWarmupIteration, Mono<POOLABLE> allocator) {
```

Comment:
how about keeping this API as before and trigger drain as suggested above

Reply:
Applied feedback, see applied feedback, see https://github.com/reactor/reactor-pool/pull/171/commits/c6ddf8b593f8504711bccaa51484e8761c48c317